src/test/java/salesforcetest/steps/AccountSteps.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package salesforcetest.steps;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+import salesforce.entities.Account;
+import salesforce.entities.Context;
+import salesforce.api.AccountHelper;
+import salesforce.utils.SheetManager;
+
+import io.cucumber.java.en.Given;
```

Comment:
imports are not necessary separate

please press ctrl + alt + O to the identation of the comments